CONTRIBUTING.rst

```diff
@@ -75,20 +75,22 @@ Generally the more people that can look over a pull request the better it will t
 
 .. _pull requests: https://github.com/sunpy/sunpy/pulls
 
-Code
-----
+Code (to solve Issues)
```

Comment:
What if it's not an issue but the code is useful?

Reply:
Yeah I am not sure we need this caveat here either.

Reply:
All the text below talks about working on issues which I thought was on purpose. If we want to expand the scope of this section, that's fine.

Reply:
Yeah, I think adding a paragraph on what if you want to just contribute code upstream too would be good.

Reply:
Done.